pkg/app/cmd.go

```diff
@@ -211,6 +211,7 @@ func (o *options) bindFlags(flags *pflag.FlagSet) {
 		defaultRunID = uuid.New().String()
 	}
 	flags.StringVar(&o.runid, "run-id", defaultRunID, "unique identifier for a kubetest2 run")
+	flags.BoolVar(&o.cpRundirToArtifacts, "cp-rundir-to-artifacts", false, `if true, copy the test binaries and run specific metadata to the ARTIFACTS`)
```

Comment:
this should be renamed? since there's no `cp` happening now

(and the related fields)

otherwise, seems fine now 🙏

Reply:
@Rajalakshmi-Girish is on medical leave for next couple of week, will relook into the PR once she is back

Reply:
Take care!!